tools/cli/commands/create.py

```diff
@@ -389,7 +389,13 @@ def ensure_disk_exists(args, gcloud_compute, disk_name, report_errors=False):
             create_disk(args, gcloud_compute, disk_name, report_errors)
         except:
             if args.zone or args.quiet:
-                raise
+                if report_errors:
```

Comment:
I don't want to bake-in assumptions based on the current structure of the code.

Right now, `report_errors` is only true when retrying a previous, failed invocation. However, that might not be the case in the future, and so I was trying to structure the code based on what the fields mean rather than the dataflow of the method.